frontend/src/Editor/LeftSidebar/SidebarDebugger/Logs.jsx

```diff
@@ -13,10 +13,10 @@ function Logs({ logProps, idx }) {
     logProps?.type === 'navToDisablePage'
       ? logProps?.message
       : logProps?.isQuerySuccessLog
-      ? 'Completed'
-      : logProps?.type === 'component'
-      ? `Invalid property detected: ${logProps?.message}.`
-      : `${startCase(logProps?.type)} failed: ${logProps?.message ?? ''}`;
+        ? 'Completed'
+        : logProps?.type === 'component'
+          ? `Invalid property detected: ${logProps?.message}.`
+          : `${startCase(logProps?.type)} failed: ${logProps?.message === undefined ? logProps?.error?.message : logProps?.message}`;
```

Comment:
Should we check for other falsy values as well.?
```
logProps?.message ? logProps?.error?.message : logProps?.message
```

Reply:
> Should we check for other falsy values as well.?
> 
> ```
> logProps?.message ? logProps?.error?.message : logProps?.message
> ```

yes, we can check for other falsy values also like this
logProps?.message ? logProps?.message : logProps?.error?.message